docs/sources/reference/components/otelcol/otelcol.processor.groupbyattrs.md

```diff
@@ -0,0 +1,88 @@
+# otelcol.processor.groupbyattrs
+
+`otelcol.processor.groupbyattrs` accepts telemetry data from other `otelcol`
+components and reassociates spans, log records and metric datapoints to a Resource that matches with the specified attributes. It groups telemetry data by specified attributes.
+
+{{% admonition type="note" %}}
+`otelcol.processor.groupbyattrs` is a wrapper over the upstream OpenTelemetry
+Collector `groupbyattrs` processor.  If necessary, bug reports or feature requests
+will be redirected to the upstream repository.
+{{% /admonition %}}
+
+It is recommended to use the groupbyattrs processor together with batch processor, as a consecutive step, as this will reduce the fragmentation of data (by grouping records together under matching Resource/Instrumentation Library)
```

Comment:
There are probably some other changes that will be required here once the other update is applied